programmers/난이도별/level01.수박수박수박수박수박수/sokkanji.js

```diff
@@ -0,0 +1,13 @@
+function solution(n) {
+    let answer = '';
+    if(n < 10000){
+        for(let i = 0; i < n; i++){
+            if(i % 2 === 0){
+                answer+='수';
+            } else{
+                answer+='박';
+            }
+        }
+    }
+    return answer;
```

Comment:
```suggestion
    return Array(n).fill(0).map((v, k) => k % 2 ? '박' :'수').join('');
```
Array의 method를 잘 활용하면
이렇게도 풀이할 수 있습니다!

Reply:
한줄로 처리하는 방법이 있었네요..!😲 코드리뷰 해주셔서 감사합니다!
덕분에 fill, map, join method에 대해 배워갑니다!